product/roundhouse.tests/infrastructure/filesystem/WindowsFileSystemAccessSpecs.cs

```diff
@@ -49,13 +50,20 @@ public class when_reading_files_with_different_formats : concern_for_file_system
             [Observation]
             public void utf8_encoded_file_should_read_correctly()
             {
-                utf8_file.should_be_equal_to("INSERT INTO [dbo].[timmy]([value]) VALUES('G�')");
+                utf8_file.should_be_equal_to("INSERT INTO [dbo].[timmy]([value]) VALUES('Gã')");
             }
 
             [Observation]
             public void ansi_encoded_file_should_read_correctly()
+            {   
+                ansi_file.should_be_equal_to(to_default_codepage("INSERT INTO [dbo].[timmy]([value]) VALUES('Gã')"));
+            }
+
+            private string to_default_codepage(string englishCodepageString)

```

Comment:
Almost all good, except for the variable name. Look at the others. Use the same format. Thanks!